smaretski/03_HTML/task_01_forms.html

```diff
@@ -0,0 +1,172 @@
+		<p><form id="my_form_4" name="vote" method="get" action="http://sc.it-academy.by/fe/form.php">
+		<fieldset>
+		<legend>Как вам?</legend>
+		<input type="radio" name="radio_vote" value="vote_1" id="checkbox_vote_1" checked>ничего
+		<br>
+		<input type="radio" name="radio_vote" value="vote_2" id="checkbox_vote_2" checked>неплохо
+		<br>
+		<input type="radio" name="radio_vote" value="vote_3" id="checkbox_vote_3" checked>хорошо
+		<br>
+		<input type="radio" name="radio_vote" value="vote_4" id="checkbox_vote_4" checked>супер
+		<br>
+		<input type="submit" name="send" value="Отправить">
+		<input type="hidden" name="data_voite" value="2013.12.31">
+		</fieldset>
+		</form>
+		</p>
+		</th>
+		</tr>
+		<tr>
+		<th align="left" valign="top">
+		<b>Опрос (голосование):</b>
+		<p><form id="my_form_5" name="feedback" method="get|post" action="http://sc.it-academy.by/fe/form.php">
+		<fieldset>
+		<legend>Оставьте Ваш отзыв</legend>
+		<input type="text" name="login" required><span>*</span>Имя
+		<br>
+		<br>
+		<input type="email" name="email" required><span>*</span>e-mail
+		<br>
+		<textarea name="feedback" placeholder="Форма отправки"></textarea>
+		<input type="submit" name="send" value="Отправить">
+		</fieldset>
+		</form>
+		</p>
+		</th>
+		<th align="left" valign="top">
+		<b>Выпадающий и мульти списки</b>
+		<p><form id="my_form_6" name="list_city" method="get" action="http://sc.it-academy.by/fe/form.php">
+		<fieldset>
+		<legend>Города РБ</legend>
+		<select name="list" id="list_blr"> 
+		<option value="0">Брест</option>	
+		<option value="1">Витебск</option>
+		<option value="2">Гомель</option>
+		<option value="3">Гродно</option>
+		<option value="4">Минская обл.</option>
+		<option value="5">Могилев</option>
+		<option value="6">Минск</option>
+		</select>
+		<select name="list[]" size="8" id="list_blr_ru" multiple>
+		<optgroup label="Города РБ">
+		<option value="1">Брест</option>
+		<option value="2">Минск</option>
+		<option value="3">Витебск</option>	
+		</optgroup>
+		<optgroup label="Города России">
+		<option value="4">Москва</option>
+		<option value="5">Рязань</option>
+		<option value="6">Магадан</option>
+		</optgroup>	
+		</select>
+		<input type="submit" name="send" value="OK">
+		</fieldset>
+		</form>
+		</p>
+		</th>
+		</tr>
+		<tr>
+		<th align="left" valign="top">
+		<b>Загрузка файла на сервер</b>
+		<p><form id="my_form_7" name="upload" method="post" enctype="multipart/form-data" action="http://sc.it-academy.by/fe/form.php">
+		<fieldset>
+		<legend>Upload</legend>
+		Файл:
+		<br>
+		<input type="file" name="file_field" dir="ltr">
+		<br>
+		Срок хранения (года):
+		<br>
+		<input type="text" name="name_01" pattern="[0-9]{4}">
+		<input type="submit" name="download" value="загрузить">
+		</fieldset>
+		</form>
+		</p>
+		</th>
+		<th align="left" valign="top">
+		<b>Валидация данных HTML5:</b>
+		<p><form id="my_form_8" name="html5" method="get" action="http://sc.it-academy.by/fe/form.php">
```

Comment:
remove p

Reply:
do not put tag form inside p.
p - for human text should be paragraph with text

Reply:
Fixed. I removed tag <p> in file